src/state-summary/state-card-climate.html

```diff
@@ -74,7 +74,9 @@
 
     if (stateObj.attributes.operation_mode === 'off') {
       stateTemp = '';
-    } else if (stateObj.attributes.operation_mode === 'auto') {
+    } else if (stateObj.attributes.operation_mode === 'auto' &&
```

Comment:
I've said this before, but why even check for `auto`? Just check if we have a range, if so: output a range, otherwise output the normal temperature.